graphql-compiler-plugin-tests/src/test/resources/ballerina_sources/valid_service_1/service.bal

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/graphql;
+
+type Person record {
+    string name;
+};
+
+type Student record {
+    int age;
+};
+
```

Comment:
Shall we add a resource returning an enum as well? It should be valid.

Reply:
Please review d9cb8fd3fd2c7deea6793f027082ac0de45dabb1